build/tasks/ProcessSharedFrameworkDeps.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
A few targets were copied from https://github.com/dotnet/core-setup/tree/be8d8e3486b2bf598ed69d39b1629a24caaba45e/tools-local/tasks. We'll need to keep these files in sync. Should we look for ways to share this code potentially?

Reply:
Yes, absolutely yes. Until then, add this comment into the affected files so we know where to look to update things.